templates/sightings.html

```diff
@@ -4,6 +4,20 @@
 
 {% block body %}
 
-<!-- Posts of various monsters containing an image, header, and caption -->
+<h1>Sightings</h1>
+<form>
```

Comment:
for your form to work, you need an action="" with a /something that links to a @app.get("/something") or @app.post("/something")